packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -971,6 +979,13 @@ export type IDirectiveResolvers${contextType} = ${name}<ContextType>;`
       });
       const mappedTypeKey = isSubscriptionType ? `${mappedType}, "${node.name}"` : mappedType;
 
+      const directiveMappings =
+        node.directives
+          ?.map(directive => this._customDirectiveToResolverTypeNameMappings.get(directive.name as any))
```

Comment:
Something is wrong with the typings of `node`. The type of directive indicates accessing via `directive.name.value`, but actually the `directive.name` is already the string at runtime.